### DISCUSSION TITLE
Getting everyone up to speed...

### DISCUSSION THREAD

**Discussion by Kernix13** [jdwilkin4/contrast-ratio-repo#13] [General]
How many people are interested in this project but who are not engaged in the Discord server? It seems like a lot. We need to walk people step by step through the git process and have them do 1 simple thing like a code block in index.html. Installing git on your computer can have stumbling blocks, then cloning, creating a branch, making changes, pushing...it can be overwhelming. Should we create a poll to see if there are reasons why some people are not joining in on discord?

**Comment by jdwilkin4** [comment]
Everyone that is listed on the developer list for this project has git installed. 
The people who are new to git and the whole process have tickets assigned to them to learn how to create their first branch, push up changes, and create their first PR.

> Should we create a poll to see if there are reasons why some people are not joining in on discord?
Everyone on the developer list is on discord already

For those who responded to the original forum post but decided not to join the project, might have decided against participating. 
That is common for Open source projects. 
People will express interest but things come up or people are no longer able to participate for a variety of reasons. 
If someone joins the discord and reaches out about contributing, then we can get them setup and hand them a `good first issue`.

**Reply by Kernix13** [reply]
Got it. I just noticed that there was myself and only 2-3 other people involved.

**Comment by AndreyDmitriev39r** [comment]
How do we feel about adding more guys from FCC community to the team in general?
One of them asked me about possible collaboration (not about our project, just in general) couple of days ago.
I mentioned our project, but we were setting up stuff back then, I didn't know: are we adding people or not. So I didn't make any suggestions to join.
Maybe I should ask directly about participation now in this case?
Or we are waiting before people reach out first and ask direct questions about joining?

**Reply by Kernix13** [reply]
I think multiple small groups would be better than 1 huge group. I'm sure we have some people who are green when it comes to JavaScript and/or using Git and creating branches. For those people, creating simple repos and going slow would be best. As a matter of fact, a repo on Git, Markdown, or CSS animations would get them comfortable with the process. 

Imagine 10-20 people waiting to grab JS features and cutting out people who are in different time zones. Have them join the discord channel so everyone can chime in on what they think. Personally, I would prefer to work on JS but I only have pushes in my profile. Doing code reviews for HTML and CSS would look great for me as well as for other people. I don't feel that I am expert at CSS, but I don't think I'm good enough to do code reviews for JS.

It would also be good if we could group people geographically so that everyone has availability at the same time. 

Another repo idea is to create a repo on how to create a Git/GitHub collaboration group. It would probably only be markdown files but it would create a list of To-Dos and steps to follow.

**Reply by AndreyDmitriev39r** [reply]
>> I think multiple small groups would be better than 1 huge group. I'm sure we have some people who are green when it comes to JavaScript and/or using Git and creating branches. For those people, creating simple repos and going slow would be best. As a matter of fact, a repo on Git, Markdown, or CSS animations would get them comfortable with the process.
>
I agree.

> It would also be good if we could group people geographically so that everyone has availability at the same time.
> 
Yeah, timezones may be a problem. Tho I think it's useful experience - to manage collaboration when dealing with different timezones
